releasenotes/notes/37415.yaml

```diff
@@ -6,3 +6,4 @@ issue:
 releaseNotes:
 - |
   **Added** a new analyzer for envoy filter patch operations to address issue 37415
+  **Updated** a new analyzer for envoy filter patch operations to address issue 37415
```

Comment:
Can we update the lines to not say #37415 and have some descriptive text. There will be a link that points to the issue added in the releases notes.

Reply:
And perhaps updating the original release notes `**Added**` item instead of added an additional `**Updated**` line.  Since the code changes will still be in the same release it reads a little funny that something was `Added` and `Updated` in the same note.  Also, not sure if the `gen-release-notes` handles multiple releaseNotes lines